### PR TITLE
chore: rename `master` branch references to `main` branch

### PR DESCRIPTION
**Pull request by emyarod**
Since we have some references and links to files on Carbon and Carbon website's `master` branch, this PR updates those references to point to `main` branch now

#### Testing / Reviewing

Confirm the updated references point to the correct resources

### CODE REVIEW

**Review by jillianhowarth — COMMENTED** [review]
This is great! There are a few 404s and links that don't work, but I'm not sure if those are from the change or if they didn't point to the right place.

File: src/pages/developing/angular-tutorial/step-4.mdx [hunk 1]
```diff
@@ -565,7 +563,7 @@ Once you save, go ahead and resize your browser to see the responsive layout at
 the different breakpoints. Make sure to review these color and spacing tokens.
 There are also a few breakpoint mixins that may be new to you. The
 `@carbon/layout`
-[SassDoc](https://github.com/carbon-design-system/carbon/blob/master/packages/layout/docs/sass.md)
+[SassDoc](https://github.com/carbon-design-system/carbon/blob/main/packages/layout/docs/sass.md)
```
[the remark this answers is not in the document]
Comment (emyarod) [reply src/pages/developing/angular-tutorial/step-4.mdx:566]: looks like this was removed in https://github.com/carbon-design-system/carbon/pull/7710

File: src/pages/developing/react-tutorial/step-4.mdx [hunk 2]
```diff
@@ -354,7 +354,7 @@ Once you save, go ahead and resize your browser to see the responsive layout at
 the different breakpoints. Make sure to review these color and spacing tokens.
 There are also a few breakpoint mixins that may be new to you. The
 `@carbon/layout`
-[SassDoc](https://github.com/carbon-design-system/carbon/blob/master/packages/layout/docs/sass.md)
+[SassDoc](https://github.com/carbon-design-system/carbon/blob/main/packages/layout/docs/sass.md)
```
[the remark this answers is not in the document]
Comment (emyarod) [reply src/pages/developing/react-tutorial/step-4.mdx:357]: looks like this was removed in https://github.com/carbon-design-system/carbon/pull/7710

File: src/pages/developing/vue-tutorial/step-4.mdx [hunk 3]
```diff
@@ -435,7 +435,7 @@ Once you save, go ahead and resize your browser to see the responsive layout at
 the different breakpoints. Make sure to review these color and spacing tokens.
 There are also a few breakpoint mixins that may be new to you. The
 `@carbon/layout`
-[SassDoc](https://github.com/carbon-design-system/carbon/blob/master/packages/layout/docs/sass.md)
+[SassDoc](https://github.com/carbon-design-system/carbon/blob/main/packages/layout/docs/sass.md)
```
[the remark this answers is not in the document]
Comment (emyarod) [reply src/pages/developing/vue-tutorial/step-4.mdx:438]: looks like this was removed in https://github.com/carbon-design-system/carbon/pull/7710

**Review by jillianhowarth — APPROVED** [review]
LGTM!

**Review by tw15egan — APPROVED** [review]
LGTM 👍 ✅

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**